sdk/nextjs/src/common/transformConfig.ts

```diff
@@ -0,0 +1,40 @@
+import { ConfigBody } from '@devcycle/types'
+
+/**
+ * Transform the config to ensure all dates are valid Date objects
+ * @param configData
+ * @returns
+ */
+export const transformConfig = (configData: ConfigBody): ConfigBody => {
```

Comment:
the input arg here is not a ConfigBody since it hasn't been transformed into one yet